hazardous/_survival_boost.py

```diff
@@ -485,11 +483,23 @@ def predict_cumulative_incidence(self, X, times=None):
             \sum_{k=1}^K \mathbb{P}(T^* \leq t \cap \Delta = k)
             + \mathbb{P}(T^* > t) = 1
 
+        Parameters
+        ----------
+        X : array-like of shape (n_samples, n_features)
+            The input samples.
+
+        times : array-like, default=None
+            The time horizons at which to estimate the probabilities. If `None`, it uses
+            the grid generated during `fit` based on the parameter `n_time_grid_steps`.
+
         Returns
         -------
-        predicted_curves : array-like of shape (n_samples, n_events + 1)
-            The first column holds the survival probability to any event and others the
-            incicence probabilities for each event.
+        predicted_curves : ndarray of shape (n_samples, n_events + 1, n_times)
+            The estimated probabilities at different time horizons. The column
```

Comment:
"column" is not meaningful any more for a 3D array.

The values for event with index 0 are the estimated probabilities of staying event-free at the requested time horizons for each observation described by the matching row of X. ...